infrastructure/cdn-in-a-box/traffic_portal_integration_test/Dockerfile

```diff
@@ -47,7 +47,6 @@ RUN npm install --save-dev jasmine-reporters@^2.0.0
 RUN ls -lah /lang/traffic_portal/test/end_to_end
```

Comment:
Can't write on the relevant line, but on 37 that should probably also be changed to node 12.